src/less/core/typography.less

```diff
@@ -17,86 +15,68 @@ h1, h2, h3, h4, h5, h6,
 
     .material-icons {
         display: inline-flex;
-        line-height: @headings-line-height;
         vertical-align: text-top;
     }
 }
 /* stylelint-enable selector-list-comma-newline-after */
 
 h1,
 .h1 {
-    font-size: 2.5rem;
+    font-size: 24px;
```

Comment:
This won't change if user has a custom browser font-size. Use rem (1.5rem)?
This also applies to the rest of units set with pixels.